luxjs/forms/form.js

```diff
@@ -677,7 +677,13 @@
                             if (isString(message))
                                 message = {message: message};
 
-                            var field = message.field || formDefaults.FORMKEY;
+                            var field = message.field;
+                            if (field && !scope[scope.formName].hasOwnProperty(field)) {
+                                message.message = field + ' ' + message.message;
+                                field = formDefaults.FORMKEY;
+                            } else if (!field) {
+                                field = formDefaults.FORMKEY;
+                            }
 
```

Comment:
Avoids JavaScript error if the API returns an error about a field that's not in the form. And displays the error so there's some indication something is wrong.